substratevm/src/com.oracle.svm.test.jdk11/src/com/oracle/svm/test/ReusePortAvailableTest.java

```diff
@@ -0,0 +1,21 @@
+package com.oracle.svm.test;
```

Comment:
Move the test into package `com.oracle.svm.test.jdk11` to prevent `mx canonicalizeprojects` issues.